pkg/extract/extract.go

```diff
@@ -101,7 +103,14 @@ func Unzip(archive, target string) error {
 	}
 
 	for _, file := range reader.File {
+		/* #nosec */
```

Comment:
It's odd to have to disable the security check, even if you added a fix for it.
This could be `/* #nosec G305 */` to keep other checks enabled.